### DISCUSSION TITLE
Skipped Test Cases in Allure Reports

### DISCUSSION THREAD

**Discussion by driesie** [allure-framework/allure2#2329] [Questions & Support] [answered]
How do I show test cases as skipped in Allure Reports?

I'm using AllureLifecycle to update my test cases but I can't seem to update the status.

I'm updating parameters and descriptions like the below:

`lifecycle.updateTestCase(testResult -> testResult.setParameters(parameters));
 lifecycle.updateTestCase(testResult -> testResult.setDescriptionHtml(testCaseDescription));`
        
  But the below doesn't seem to update the test case status to 'Skipped'
  
   `lifecycle.updateTestCase(testResult -> testResult.setStatus(Status.SKIPPED));`    
   
   What am I missing?

**Comment by baev** [comment, marked as answer]
Allure doesn't change the test execution. Think of Allure as a logger to enrich your reports with additional metadata. 

The test framework always sets the test status after the test is executed. So, any value you set in runtime will be overridden by the test framework. The solution would be to skip the test. You can do this by throwing `SkipException` for TestNg or using the `Assumptions ` helper for JUnit.

However, there is a possibility to modify the test results after the test is finished by using the `TestLifecycleListener` extension point provided by Allure Java SDK.

**Reply by driesie** [reply]
Perfect thank you I see now where my understanding was incorrect.

I managed to flag the tests as Skipped by using the below based of a parameter that will instruct the test to be executed or skipped.

`Assume.assumeTrue("Skipping test due to instruction from CSV action field.", false);`
